promgen/templates/promgen/vue/silence_create_modal.html

```diff
@@ -56,10 +56,10 @@ <h5>
                   <li v-for="(value, label) in state.labels">
                     <div class="promgen-label-target">
                       <span
-                        :title="`${label}:${value}`"
+                        :title="`${label}${value[1]}${value[0]}`"
                         class="promgen-silence-matcher-truncate"
                       >
-                        [[label]]=[[value]]
+                        [[`${label}${value[1]}${value[0]}`]]
```

Comment:
In the `silence_list_modal.html` file, line number 63, we are doing this:

```
[[item.label]][[item.operator]][[item.value]]
```

What is the reason for using the backtick (`) operator to create a template literal in here? Is it because this code didn't work?

```
[[ label ]][[ value[1] ]][[ value[0] ]]
```

Maybe there is a problem with the brackets :shrug: Could you confirm?

Reply:
Fixed in 0b14376bbd7f5284e19e6551c3b879fc443bfb0a.